### DISCUSSION TITLE
Destination drive full

### DISCUSSION THREAD

**Discussion by vvavepacket** [ericaltendorf/plotman#499] [Q&A]
Currently in not using archiving and setting only my destination drive..

What happens if my destination drive is full. Does plotman crashes?

Or  everything just stop at phase 4 and will be stick on there untilI free some space? Any thoughts?

**Comment by cyperbg** [comment]
Good question. Following.

**Reply by cr1za** [reply]
I had assumed plotman checks for drive space before starting a new plot to that destination file, but now I noticed I have four plots running to one of my destination drives and there's only free space left for one plot on that drive....

**Reply by cyperbg** [reply]
Quickly remove some plots to another external drive and edit the config file to reflect the changes.
You need to manually calculate how many you can fit.
Or use this: https://plot-plan.chia.foxypool.io/
